mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java

```diff
@@ -121,10 +121,9 @@ public void hideProgress() {
     }
 
     public void hideKeyboard(View view) {
-        InputMethodManager inputManager = (InputMethodManager) this
-                 .getSystemService(Context.INPUT_METHOD_SERVICE);
-        inputManager.hideSoftInputFromWindow(getWindow().getAttributes().token, InputMethodManager
-                 .RESULT_UNCHANGED_SHOWN);
+        InputMethodManager inputMethodManager = (InputMethodManager)
+                getSystemService(Context.INPUT_METHOD_SERVICE);
+        inputMethodManager.hideSoftInputFromWindow(getCurrentFocus().getWindowToken(), 0);
```

Comment:
Hey @robustTechie. Using this line of code causes [this](https://github.com/openMF/android-client/issues/1219) issue. I had fixed it in [this](https://github.com/openMF/android-client/issues/1219) PR. What you're doing right now will cause the issue to occur again. Leaving this block of code as it was will work the same way, I guess.

Reply:
After adding this code the keyboard is dismissed if I open the navigation drawer without causing any application crash.